gslib/utils/system_util.py

```diff
@@ -250,11 +249,13 @@ def IsRunningInteractively():
 
 
 def MonkeyPatchHttp():
-  if platform.python_version().startswith('3.6.5'):
-    _MonkeyPatchHttpForPython_3_6_5()
+  ver = sys.version_info
+  # Checking for and applying monkeypatch to python versions 3.4 - 3.6.7
+  if (ver.major == 3 and (ver.minor < 6 or (ver.minor == 6 and ver.micro < 8))):
```

Comment:
Nit: `ver.micro < 8` should be `ver.micro < 7`, since 3.6.7 prints the header values.

Reply:
Changed!